src/js/bootstrap-dialog.js

```diff
@@ -1210,7 +1190,7 @@
             this.getModalHeader().append(this.createHeaderContent());
             this.getModalBody().append(this.createBodyContent());
             this.getModal().data('bs.modal', new BootstrapDialogModal(this.getModalForBootstrapDialogModal(), { //FIXME for BootstrapV4
-                backdrop: 'static',
+                backdrop: this.canCloseByBackdrop() ? true : 'static',
```

Comment:
Now when I think of it, perhaps this line should be:
```suggestion
                backdrop: (this.isClosable() && this.canCloseByBackdrop()) ? true : 'static',
```

I've never used non-closable dialogs, and I currently don't have an environment to check. Perhaps you can check it as a part of your tests.